lib/shopify_transporter/exporters/magento/soap.rb

```diff
@@ -7,6 +7,12 @@ module ShopifyTransporter
   module Exporters
     module Magento
       class Soap
+        class FailedLoginError < Exporters::ExportError
+          def initialize(error_message)
+            super("Unable to obtain SOAP session ID from server.#{$/ + $/}Details:#{$/ + $/ + error_message}")
```

Comment:
Should we have a test showing the exception's response and how it's formatted (I believe, we have the same sorts of tests for other exceptions, outputs)

Reply:
👍 to Fahia's suggestion